src/dlls/mscoree/unixinterface.cpp

```diff
@@ -128,7 +130,7 @@ static void ConvertConfigPropertiesToUnicode(
     for (int propertyIndex = 0; propertyIndex < propertyCount; ++propertyIndex)
     {
         propertyKeysW[propertyIndex] = StringToUnicode(propertyKeys[propertyIndex]);
-        propertyValuesW[propertyIndex] = StringToUnicode(propertyValues[propertyIndex]);
+        propertyValuesW[propertyIndex] = propertyValues[propertyIndex] != nullptr ? StringToUnicode(propertyValues[propertyIndex]) : nullptr;
```

Comment:
Does this mean we try to actually convert the bundle probe pointer to UTF16 string? Isn't that like super dangerous...